gui/src/context/taipyReducers.ts

```diff
@@ -673,18 +673,19 @@ export const createTimeZoneAction = (timeZone: string, fromBackend = false): Tai
 });
 
 const getAlertType = (aType: string) => {
-    aType = aType.trim() || "i";
-    aType = aType.substr(0, 1).toLowerCase();
+    aType = aType.trim();
+    aType = aType && aType.charAt(0).toLowerCase();
     switch (aType) {
         case "e":
             return "error";
         case "w":
             return "warning";
         case "s":
             return "success";
-        default:
+        case "i":
             return "info";
     }
+    return aType;
```

Comment:
So the user can provide his own alert type?

Reply:
no, I'll change this